### PR TITLE
feat: Add catalan support

### PR DESCRIPTION
**Pull request by markusand**
Catalan is a language spoken by over 9 million people in Catalonia, the Balearic Islands, and Valencia, in the south of France, in the Principality of Andorra, and in the Alghero locality in Italy. Recognizing it in software and platforms acknowledges the linguistic and cultural significance of a vast community.

### CODE REVIEW

File: localization/languages/ca.json [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    "passwordManagerSetupUnlockError": "No s'ha pogut desbloquejar la botiga de contrasenyes: ",
+    "passwordManagerSetupRetry": "Assegura't d'utilitzar el fitxer correcte i d'introduir una contrasenya correcta. Pots tornar a intentar-ho arrossegant l'eina aquí de nou.",
+    "passwordManagerUnlock": "Introdueix la teva contrasenya per utilitzar l'autocompletat:",
+    "passwordManagerUnlocking": "Desbloquejant...",
+    "passwordManagerUnlockError": "Hi ha hagut un error desbloquejant la botiga de contrasenyes.",
+    "passwordManagerEnable": "Habilita el suport de Bitwarden a les preferències per començar a utilitzar l'autocompletat.",
+    "passwordManagerGetStarted": "Per començar a utilitzar l'autocompletat, ",
+    "passwordManagerGoToSettings": "ves a les preferències",
+    "passwordManagerSetup": "Configura un gestor de contrasenyes",
+    "passwordManagerUnlockErrorRetry": "Torna a intentar-ho"
```
Comment (PalmerAL) [inline localization/languages/ca.json:259]: These don't match the strings listed at the end of the English source: https://github.com/minbrowser/min/blob/master/localization/languages/en-US.json - where did they come from?

Comment (markusand) [reply localization/languages/ca.json]: TBH I don't know. I've checked again en-US and also found few missing keys.

Comment (PalmerAL) [reply localization/languages/ca.json]: Thanks for updating. Just to confirm, did you write all these translations yourself, or are they machine translated (eg chatGPT)?

Comment (markusand) [reply localization/languages/ca.json]: I used automatic translation and then skimmed the result for inconsistencies (verb tense, formal/informal use, etc.). At first glance, it looked good to me, but it's pretty obvious I missed some things. Let me doublecheck one by one to be sure everything fits the quality standards it deserves.

Comment (PalmerAL) [reply localization/languages/ca.json]: OK, sounds good, once you're done I'll re-review.

Comment (markusand) [reply localization/languages/ca.json]: I've double checked and applied few fixes to use a more natural language.

File: localization/languages/ca.json [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    "passwordManagerSetupLinkInstaller": "l'instal·lador CLI de %p",
+    "passwordManagerSetupStep2": "Després, arrossega l'eina a la caixa de sota:",
+    "passwordManagerSetupDragBox": "Arrossega l'eina aquí",
+    "passwordManagerSetupInstalling": "Instal·lant...",
+    "passwordManagerBitwardenSignIn": "Per connectar el teu compte Bitwarden, ves a vault.bitwarden.com/#/settings/account, desplaça't fins al final, i tria \"Veure clau API\". Després, enganxa els valors als camps de sota.",
+    "passwordManagerSetupSignIn": "Inicia sessió al teu gestor de contrasenyes per començar a utilitzar l'autocompletat. Les teves credencials no es desaran enlloc dins de Min.",
+    "disableAutofill": "Deshabilitar autocompletat",
+    "passwordManagerSetupUnlockError": "No s'ha pogut desbloquejar el magatzem de contrasenyes: ",
+    "passwordManagerSetupRetry": "Assegura't d'utilitzar el fitxer correcte i d'introduir una contrasenya correcta. Pots tornar a intentar-ho arrossegant l'eina aquí de nou.",
+    "passwordManagerUnlock": "Introdueix la teva contrasenya per utilitzar l'autocompletat:",
```
Comment (PalmerAL) [inline localization/languages/ca.json:276]: This has a parameter for the manager name in the original:

https://github.com/minbrowser/min/blob/6ab87e48f82aebf215e268910a4635130ffd4243/localization/languages/en-US.json#L279

**Comment by PalmerAL** [comment]
Thanks!

**Pull request merged**